setup.py

```diff
@@ -74,13 +74,27 @@ def list_modules(dirname):
 
 
 def load_description():
+    in_patrons = False
     in_raw = False
 
     description_lines = []
 
     # NOTE(kgriffs): PyPI does not support the raw directive
     for readme_line in io.open('README.rst', 'r', encoding='utf-8'):
-        if readme_line.startswith('.. raw::'):
+
+        # NOTE(vytas): The patron list largely builds upon raw sections
+        if 'big thank you to our patrons' in readme_line.strip().lower():
```

Comment:
One thing that might make parsing a bit simpler is to add some cross-reference bookmarks to `README.rst`. They would not influence the normal rendering on GitHub, but you could check for them here as a way of knowing what sections to leave out, e.g.:

```rst
.. _begin_patrons:

blah blah

.. _end_patrons:
```

Reply:
I have now hopefully improved the data and code separation in that vein, even though it is still somewhat awkward... but maybe it is in the nature of this substitution itself.

Reply:
Thanks, I think this is sufficient for now.